docs/INSTALL.md

```diff
@@ -11,9 +11,50 @@ Before you begin the installation process, please make sure you have all the bhi
  2. [Redis](redis.io)
  3. [NodeJS](https://nodejs.org/en/) (we recommend using [node version manager](https://github.com/creationix/nvm) on linux.  Note that we only test on stable and edge).
  4. [WKHTMLtoPDF](http://wkhtmltopdf.org/downloads.html) (use the compiled binaries, even if it is distributed with your package manager.  The binaries come with patched Qt).
+ 5.	npm
+ 6.	curl
+ 7.	yarn
+ 8.	git
+ 
+###### Detailed dependency installation instructions for Ubuntu (verified / installed specifically using VirtualBox)
+
+```bash
+#Run the following command to update the package lists:
+sudo apt-get update
+
+#Install MySQL with the following command:
+sudo apt-get install mysql-server
+
+#Run the following commands to install Redis:
+sudo apt-get install redis-server
+
+#Run the following command to install NodeJS:
+sudo apt-get install nodejs
+
+#Run the following commands to install WKHTMLtoPDF:
+sudo apt-get install xvfb
+wget https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.4/wkhtmltox-0.12.4_linux-generic-amd64.tar.xz
+tar xvf wkhtmltox-0.12.4_linux-generic-amd64.tar.xz
+sudo mv wkhtmltox/bin/wkhtmltopdf /usr/bin 
+sudo rm wkhtmltox-0.12.4_linux-generic-amd64.tar.xz  && rm -rf wkhtmltox
+
+#Run the following commands to install npm:
+sudo apt-get install npm
```

Comment:
If we use the `nvm` instructions, `npm` will automatically be downloaded and usable from the command line.